### ISSUE TITLE
New Analyzer: Symantec Content Analysis 

### ISSUE THREAD

**Issue by yesOrMaybeWhatever** [TheHive-Project/Cortex-Analyzers#380]
### Request Type

 Analyzer

### Description
Add Symantec Content Analysis. 

https://support.symantec.com/en_US/article.DOC10634.html

**Comment by nadouani** [comment]
This analyzer needs to be contributed by the community because, we (core team) don't have access to a Symantec Content Analysis sandbox. If you have it, we can help you write the analyzer, or if someone else have it and can contribute than it's more than welcome

Thanks

**Comment by yesOrMaybeWhatever** [comment]
@nadouani Yes, we have one in our environment. "we can help you write the analyzer" how does this help look like ?

**Comment by nadouani** [comment]
An analyzer is a simple Python script usually calling some APIs. 

First question: does that product (SCA) expose a HTTP API to submit files for analysis?
We already made that with Joe Sandbox and we can make the same work.

**Comment by yesOrMaybeWhatever** [comment]
@nadouani please have a look at https://support.symantec.com/en_US/article.DOC10465.html I guess this is what you are asking about :)

**Comment by yesOrMaybeWhatever** [comment]
@nadouani Could you please assist me ? I look forward to your answer. Help is much appreciated!
